portfolio/src/main/webapp/mocha_tests/calendar_test.js

```diff
@@ -0,0 +1,131 @@
+var assert = require('assert');
+var calendar = require('../calendar.js'); 
```

Comment:
It's usually good practice to put the _test.js file next to the .js file. But it's an optional suggestion.

Reply:
Updated.